test/buildah-bud/apply-podman-deltas

```diff
@@ -263,18 +263,28 @@ skip_if_remote "different error messages between podman & podman-remote" \
 ###############################################################################
 # BEGIN temporary workarounds that must be reevaluated periodically
 
-skip_if_remote "FIXME: can this test be made to work under podman-remote?" \
-               "bud-with-mount-cache-like-buildkit-verify-default-selinux-option"
-
-skip_if_rootless_remote "FIXME: #17788 tar + namespaces over http" \
-                        "bud-http-context-with-Dockerfile" \
-                        "bud-http-context-dir-with-Dockerfile" \
-                        "bud-http-context-with-Containerfile"
-
-skip_if_rootless_remote "FIXME: not sure if 17788 or some other bug" \
-                        "bud-github-context" \
-                        "bud with Dockerfile from stdin tar" \
-                        "build-with-network-test"
+#skip_if_remote "FIXME: can this test be made to work under podman-remote?" \
+#               "bud-with-mount-cache-like-buildkit-verify-default-selinux-option"
+
+#skip_if_rootless_remote "FIXME: #17788 tar + namespaces over http" \
+#                        "bud-http-context-with-Dockerfile" \
+#                        "bud-http-context-dir-with-Dockerfile" \
+#                        "bud-http-context-with-Containerfile"
+
+#skip_if_rootless_remote "FIXME: not sure if 17788 or some other bug" \
+#                        "bud-github-context" \
+#                        "bud with Dockerfile from stdin tar" \
+#                        "build-with-network-test"
+
+# 2023-06-27 UPDATE: this seems to be fixed? I think we can remove this skip?
+# 2023-05-04 fails with:
+#   Error: creating build container: initializing source docker://debian:testing-slim: reading manifest testing-slim in quay.io/libpod/debian: manifest unknown
+#skip_if_remote "FIXME: 2023-05-04: new code, fails in podman-remote" \
+#               "bud-verify-if-we-dont-clean-prexisting-path"
+
+# 2023-06-13 buildah 4746 changed exit code & expected error message
```

Comment:
ARGH! I'm sorry, this is my fault. You caught the treadmill in a bad moment, when I was seeing if any of these bugs still exist. They do. Please grab a fresh copy of the treadmill.